podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Script : MonoBehaviour
+{
+    int x;
```

Comment:
We use English name of variables and functions, please look to ours code and stay in the same pattern.